File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio07.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio07 {
    public static void main(String[] args) {
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira um número real: ");
        float x = entrada.nextFloat();
        float dobro = x * 2;
        float tercaParte = x / 3;
        System.out.println("O dobro de " +x+ " é: " +dobro+ "\nA terça parte de " +x+ " é: " +tercaParte);
    }
}

File: guanabara-c/lista01/exercicio07.c
#include <stdio.h>

int main(){
    int x, dobro_x;
    float terca_parte_x;

    printf("Insira um número: ");
    scanf("%d", &x);

    dobro_x = x * 2;
    terca_parte_x = (float)x / 3;
    printf("O dobro de %d é: %d. A terça parte de %d é %.2f.", x, dobro_x, x, terca_parte_x);

    return 0;
}
